assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -543,6 +543,102 @@ describe( 'core/modules modules', () => {
 			} );
 		} );
 
+		describe( 'getModuleDependencyNames', () => {
```

Comment:
@ivankruchkoff let's combine `getModuleDependencyNames` and `getModuleDependantNames` into one [`describe.each`](https://jestjs.io/docs/en/api#describeeachtablename-fn-timeout) block:

```js
describe.each( [
	[ 'getModuleDependencyNames', 'dependencies' ],
	[ 'getModuleDependantNames', 'dependants' ],
] )( '%s', ( selector, collectionName) => {
	...

	it( 'returns dependency module names when modules are loaded', async () => {
		fetchMock.getOnce(
			/^\/google-site-kit\/v1\/core\/modules\/data\/list/,
			{ body: FIXTURES, status: 200 }
		);
		const slug = 'optimize';
		registry.select( STORE_NAME )[ selector ]( slug );

		// Wait for loading to complete.
		await untilResolved( registry, STORE_NAME ).getModules();

		const namesLoaded = registry.select( STORE_NAME )[ selector ]( slug );

		expect( fetchMock ).toHaveFetchedTimes( 1 );
		expect( namesLoaded ).toMatchObject( fixturesKeyValue[ slug ][ collectionName ].map( ( moduleSlug ) => fixturesKeyValue[ moduleSlug ].name ) );
	} );

	...
} );
```

Reply:
Combined.

Reply:
Awesome, thanks!